web/src/routes/RepositoryDetails/TagHistory/TagHistoryToolbar.tsx

```diff
@@ -0,0 +1,93 @@
+import {
+  Checkbox,
+  SearchInput,
+  Toolbar,
+  ToolbarContent,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import {ToolbarPagination} from 'src/components/toolbar/ToolbarPagination';
+import DateTimePicker from 'src/components/DateTimePicker';
+
+export default function TagHistoryToolBar(props: TagHistoryToolBarProps) {
+  const {
+    showFuture,
+    setShowFuture,
+    query,
+    setQuery,
+    startTime,
+    setStartTime,
+    endTime,
+    setEndTime,
+    page,
+    setPage,
+    perPage,
+    setPerPage,
+    total,
+  } = props;
+  return (
+    <Toolbar>
+      <ToolbarContent>
+        <ToolbarItem variant="search-filter">
+          <SearchInput
+            placeholder="Search by name..."
```

Comment:
nit: Would `Search by tag name...` be more clear? Tested that the size of input box remains same after adding `tag` to placeholder.

Reply:
agreed, added